gcp/appengine/blog/content/posts/2023-in-review/index.md

```diff
@@ -0,0 +1,85 @@
+* [Malicious Packages](https://openssf.org/blog/2023/10/12/introducing-openssfs-malicious-packages-repository/)
+* [R](https://github.com/RConsortium/r-advisory-database)
+* [Rocky Linux](https://osv.dev/blog/posts/almalinux-and-rocky-linux-join-osv/)
+* [Haskell](https://github.com/haskell/security-advisories)
+* [Python Software Foundation](https://discuss.python.org/t/the-python-software-foundation-has-been-authorized-by-the-cve-program-as-a-cve-numbering-authority-cna/32561/3)
+
+This has brought the total number of supported ecosystems to 20. In particular, we have seen increased interest from Linux distributions, and expect to see further growth in adoption in 2024.
+
+### C/C++
+
+C and C++ dependencies remain a critical pillar of open source supply chains. We've expanded our C and C++ coverage by enriching our database with over [30,000 advisories](https://osv.dev/blog/posts/introducing-broad-c-c++-support/) with commit-level vulnerability information sourced from NVD's CVE records.
+
+Our commit level vulnerability information paired with the [experimental determineversion API](https://osv.dev/blog/posts/using-the-determineversion-api/) enables OSV-Scanner to detect vulnerabilities in C and C++ dependencies used as submodules or directly vendored into source code. Our improvements in C and C++ support closes an important gap in vulnerability detection. 
+
+## Infrastructure
+
+Behind the scenes, we've optimized OSV.dev's API for performance and reliability, to ensure we maintain our [recently defined](https://osv.dev/blog/posts/announcing-osv-service-level-objectives/) [Service Level Objectives](https://google.github.io/osv.dev/faq/#what-are-osvs-service-level-objectives-slos).
+
+We saw peaks in API usage of over 250 QPS in November.
+
+![Image shows the GitHub star history for all OSV-related GitHub repositories taken at November 17, 2023. osv-schema has approximately 150 stars, osv.dev has approximately 1,300 stars, and osv-scanner has approximately 5,400 stars.](star-history-20231117.png "GitHub star history for all OSV repos, as of 2023/11/17")
+
+## Community
+
+There’s been a very pleasing uptick in both interest and external contributions
+to both OSV.dev ([23 total contributors](https://github.com/google/osv.dev/graphs/contributors?from=2023-01-01&to=2023-12-31&type=c)) and OSV-Scanner ([32 total contributors](https://github.com/google/osv-scanner/graphs/contributors?from=2023-01-01&to=2023-12-31&type=c))
+
+We also gave an [update on OSV at the OpenSSF Day Europe](https://www.youtube.com/watch?v=WvMXsm_BEf4), in Bilbao, Spain in September.
+
+## Tooling
+Since OSV-Scanner's launch [one year ago](https://security.googleblog.com/2022/12/announcing-osv-scanner-vulnerability.html), it has grown to account for at least TBD% of OSV.dev's API traffic and now includes the following features:
```

Comment:
@oliverchang We either need to update this statistic (currently shown as TBD%) or we should cut it. Do you know what the percent should be.